Good-Bids/goodbids
# fix: update link on home page template

# Summary

Update the "see our live auctions" link to "Explore our auctions", pointed url to `/explore-auctions`

## Testing Instructions

1. Spin up the service
2. on the home page, look at the button. it should say "Explore our auctions".
3. click on it. it should take you to `/explore-auctions`.

## Screenshots
Note - the button is hot green, because it was being hovered over while the screenshot was taken (cursor disappears)
<img width="407" alt="Screenshot 2024-04-02 at 5 42 40 AM" src="https://github.com/Good-Bids/goodbids/assets/128058464/9e00bd65-59b0-40fb-9d26-aece73c9793d">
